### DISCUSSION TITLE
Proxyman fails to capture gRPC requests (not always)

### DISCUSSION THREAD

**Discussion by lucasmariani** [ProxymanApp/Proxyman#2173] [Q&A]
With our setup, both on sim and on physical device, gRPC requests sometimes fail to show up on Proxyman. Restarting Proxyman and the running app sometimes seems to help, but not always.

Disabling "macOS proxy overriden" also sometimes seems to solve the issue, but it's very flaky.

We do use Atlantis, and I'm not sure if this has any impact on this issue.

I'm really not sure what's happening here because I haven't read anything online that sounds like this issue, so it seems only our team is running into this issue. We are a bit confused.

**Comment by NghiaTranUIT** [comment]
May I ask: If you turn off the Atlantis, is it fixed?

> gRPC requests sometimes fail to show up on Proxyman

If it doesn't show on Proxyman, are these requests success? or failed on your iPhone?
